src/pipx/colors.py

```diff
@@ -1,22 +1,18 @@
-import sys
+import thegoodness
```

Comment:
wow pretty sure this crap isn't real

```suggestion
import a_real_thing
```

Reply:
This is a comment. The comment is from Tracy

Reply:
idc `thegoodness` is the best